server/src/main/java/org/cloudfoundry/identity/uaa/login/LoginInfoEndpoint.java

```diff
@@ -450,12 +451,13 @@ private void updateLoginPageModel(
             boolean fieldUsernameShow,
             boolean linkCreateAccountShow
     ) {
+        Comparator<SamlIdentityProviderDefinition> sortingByLinkText = Comparator.comparing(SamlIdentityProviderDefinition::getLinkText);
```

Comment:
This looks like it's a case-sensitive sort, right? Could we consider a case-insensitive sort here?

I'm not fluent in Java, but the docs read like it might be this simple:

```suggestion
        Comparator<SamlIdentityProviderDefinition> sortingByLinkText = Comparator.comparing(SamlIdentityProviderDefinition::getLinkText, String.CASE_INSENSITIVE_ORDER);
```

Reply:
(also thanks! this is a great improvement for our users!)

Reply:
ok